extensions/2.0/Vendor/EXT_lights_environment/README.md

```diff
@@ -0,0 +1,127 @@
+# EXT_lights_environment
+
+## Contributors
+
+* `TODO: Add contributors`
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
```

Comment:
Please add purpose of extension, along the lines of:

The purpose of this extension is to allow embedding of environment map data (IBL) in a glTF asset.
An environment map will supply diffuse and specular light contribution to the scene, as well as information needed for reflections.

This can be used on it's own - ie a glTF asset with only environment map data - for a usecase where the IBL needs to be distributed.
It can also be used together with model data, for usecases where a model shall be displayed in a defined environment.